### PR TITLE
feat: update tailwind usage in krew-angular (issue #608)

### PR DESCRIPTION
**Pull request by aaltw**
An update for better usage of tailwind:

### CODE REVIEW

File: apps/krew-ui-angular/tailwind.config.js [hunk 1]
```diff
@@ -1,18 +1,14 @@
 module.exports = {
-  prefix: '',
   mode: 'jit',
   purge: {
-    enabled: process.env.TAILWIND_MODE === 'build',
-    content: [
-      './apps/krew-ui-angular/**/*.{html,ts,css,scss,sass,less,styl}',
-      './libs/krew-angular/**/*.{html,ts,css,scss,sass,less,styl}',
-    ],
+    enabled: true,
```
Comment (aaltw) [inline apps/krew-ui-angular/tailwind.config.js:4]: Going to always purge.. editor is smart enough to read all the possible classes from tailwind and classes added by themes

File: apps/krew-ui-angular/tailwind.config.js [hunk 2]
```diff
@@ -1,18 +1,14 @@
...
+    content: ['apps/krew-ui-angular/**/*.{html,ts}', 'libs/krew-angular/**/*.{html,ts}'],
   },
   darkMode: 'media',
   theme: {
     extend: {
       colors: {
-        'brand-01': 'var(--brand-01)',
+        'brand-01': ({ opacityValue }) => `rgba(var(--brand-01), ${opacityValue})`,
```
Comment (aaltw) [inline apps/krew-ui-angular/tailwind.config.js:11]: This is a very nice addition, but requires a change in the shared vars, but those weren't used by any other product, so _dips_!

File: libs/krew-angular/auth/src/lib/login.component.html [hunk 3]
```diff
@@ -1,87 +1,5 @@
...
-  </div>
-  <div class="relative flex-1 hidden w-0 lg:block">
-    <img
-      class="absolute inset-0 object-cover w-full h-full"
-      src="https://images.unsplash.com/photo-1505904267569-f02eaeb45a4c?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1908&q=80"
-      alt=""
-    />
-  </div>
-</div>
+<div class="w-screen h-[10rem] bg-red-500/[0.12]"></div>
```
Comment (aaltw) [inline libs/krew-angular/auth/src/lib/login.component.html:1]: Removed the unused login mess, added a demonstration

File: libs/krew-angular/auth/src/lib/login.component.html [hunk 4]
```diff
@@ -1,87 +1,5 @@
...
+<div class="w-screen h-[10rem] bg-green-500/[0.12]"></div>
+<div class="w-screen h-[10rem] bg-brand-01"></div>
+<div class="w-screen h-[10rem] bg-brand-01/50"></div>
```
Comment (aaltw) [inline libs/krew-angular/auth/src/lib/login.component.html:4]: With the tailwind shorthand `/50` here, there is an opacity applied to the theme color brand-01

File: libs/krew-angular/auth/src/lib/login.component.ts [hunk 5]
```diff
@@ -1,16 +1,8 @@
...
-export class LoginComponent implements OnInit {
-  constructor() {
-    console.log('hoi');
-  }
-
-  ngOnInit(): void {
-    console.log('hoi');
-  }
-}
+export class LoginComponent {}
```
Comment (aaltw) [inline libs/krew-angular/auth/src/lib/login.component.ts:8]: Removed unneeded methods. 

Will be added later

File: libs/shared/assets/styles/theme.css [hunk 6]
```diff
@@ -1,5 +1,5 @@
 :root {
-  --brand-01: #00327f;
+  --brand-01: 0, 50, 127;
```
Comment (aaltw) [inline libs/shared/assets/styles/theme.css:2]: using the separated rgb values is much easier to use then the hex value:

in CSS you can use `color: rgb(var(--brand-01))`, yielding the same result as `color: var(--brand-01)` as it was a hex value. But the latter you can't use to apply to opacity, while the first allows you to define `color: rgb(var(--brand-01), 0.25)`, that is a 25% opacity.

**Review by mschilling — APPROVED** [review]

**Review by cassshh — APPROVED** [review]

File: apps/krew-ui-angular/tailwind.config.js (hunk 2 above)
Comment (aaltw) [reply apps/krew-ui-angular/tailwind.config.js]: Created a new issue

File: libs/shared/assets/styles/theme.css (hunk 6 above)
Comment (aaltw) [reply libs/shared/assets/styles/theme.css:2]: Moved to a new issues

**Pull request merged**